content/docs/reference/search/overview.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Search Overview
+last_edited: '2023-06-08T04:00:00.000Z'
+---
+
+TinaCMS provides built-in search functionality for content. This is useful for allowing editors to quickly finding content in a site. Tina Cloud's content search is powered by [Fergus McDowall](https://github.com/fergiemcdowall "Fergus McDowall")'s [search-index](https://www.npmjs.com/package/search-index "search-index") library.
+
+## Configuration
+
+To enable search functionality, you will need to populate the `search` field in your Tina configuration. The only required element of the search configuration when using Tina Cloud search is the `search.tina.indexerToken` field. This can be obtained from the Tina Cloud [dashboard](/docs/tina-cloud/dashboard/projects/#api-tokens) for the project.
+
+```javascript
+export default defineConfig({
+  //...
+  schema: {
+    collections: [
+      // Array of collections
+    ]
+  },
+  search: {
+    tina: {
+      indexerToken: '<Your Search Token>',
+      stopwordLanguages: ['eng']
+    },
+    indexBatchSize: 100,
+    maxSearchIndexFieldLength: 100
+  },
+  //.. Other config
+})
+```
+
+### Definition
+
+| Property          | Description                                                                                  |
+| ----------------- |----------------------------------------------------------------------------------------------|
+| `search.tina.indexerToken` | Tina Cloud search token (*required*) |
+| `search.tina.stopwordLanguages` | Optional array of string stopword languages. Defaults to `['eng']`. See the [stopword](https://github.com/fergiemcdowall/stopword#language-code) GitHub repository for a full list of supported languages. |
+| `search.indexBatchSize`   | Used by the indexing process to determine the number of documents to index per request. Defaults to `100`. |
+| `search.maxSearchIndexFieldLength` | For variable length text fields, this controls how much of the text is considered when indexing. A higher value will increase the indexing time and size of the search index. Defaults to `100` characters.|
+
+## Building the search index
+
+### Development
+
+When search is configured and a site running locally with the `dev` command, content will be automatically indexed at startup. Any changes to local content will also trigger updates to the search index.
```

Comment:
```suggestion
When search is configured and a site is running locally with the `dev` command, content will be automatically indexed at startup. Any changes to local content will also trigger updates to the (local) search index.
```

Reply:
(Just wanted to clarify that we're not updating their production search index by making local changes)